### PR TITLE
Address demo deploy restart of all apps

### PR DESCRIPTION
**Pull request by alain-hoang**

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -22,10 +25,4 @@
 # TASKS
```
Comment (jgrevich) [inline config/deploy.rb]: We can just go ahead and remove this whole section now :)

Comment (jgrevich) [inline config/deploy.rb]: Looks good, thanks @alain-hoang. Only nitpick is to remove the whole `# Tasks` section now that it's empty.

**Comment by zachmargolis** [comment]
I fixed the merge conflicts with my other changes that added back in a TASKS section

**Comment by pkarman** [comment]
@zachmargolis lgtm

please squash so there's one clean commit and then feel free to merge.

**Comment by zachmargolis** [comment]
@pkarman roger that, will squash

**Comment by pkarman** [comment]
@zachmargolis sorry, Gemfile.lock merge error now -- probably a quick fix.

**Pull request merged**